<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutoPostWidget</class>
 <widget class="QWidget" name="AutoPostWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <layout class="QHBoxLayout" name="compressPathLayout_2">
       <item>
        <widget class="QLabel" name="compressPathLbl">
         <property name="text">
          <string>compress path: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="compressPathEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="toolTip">
          <string>temporary folder where the archives and par2 will be created (it will be cleaned once the post is done)</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="compressPathButton">
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>select the temporary folder where the archives and par2 will be created (it will be cleaned once the post is done)</string>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="rarLayout">
       <item>
        <layout class="QHBoxLayout" name="rarPathLayout">
         <item>
          <widget class="QLabel" name="rarLbl">
           <property name="text">
            <string>rar path: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="rarEdit">
           <property name="toolTip">
            <string>full path of the rar executable</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="rarPathButton">
           <property name="maximumSize">
            <size>
             <width>30</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string>select rar executable</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="rarSizeLayout">
         <item>
          <widget class="QLabel" name="rarSizeLbl">
           <property name="text">
            <string>vol size:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="rarSizeEdit">
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string>to split the rar archive in several volumes (0 to don't split)</string>
           </property>
           <property name="text">
            <string>42</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_11">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="redundancyLayout">
       <item>
        <widget class="QLabel" name="redundancyLbl">
         <property name="text">
          <string>par2 redundancy (%): </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="redundancyEdit">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="toolTip">
          <string>percentage of par2 redundancy</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="SignedListWidget" name="filesList"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="autoDirLayout">
     <item>
      <widget class="QLabel" name="autoDirLbl">
       <property name="text">
        <string>&lt;b&gt;Auto Dir&lt;/b&gt; path: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="autoDirEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Auto Dir path</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="autoDirButton">
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>select the Auto Directory</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="autoDirSelectAllCB">
       <property name="toolTip">
        <string>select all files</string>
       </property>
       <property name="text">
        <string>select All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="scanAutoDirButton">
       <property name="toolTip">
        <string>Scan the Auto directory. Delete manually the files/folder you don't want to post (using DEL)</string>
       </property>
       <property name="text">
        <string> Scan</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/resources.qrc">
         <normaloff>:/icons/scanFolder.png</normaloff>:/icons/scanFolder.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="optionsLayout">
     <item>
      <layout class="QHBoxLayout" name="genNameLayout">
       <item>
        <widget class="QCheckBox" name="genNameCB">
         <property name="toolTip">
          <string>generate a random name for the archive</string>
         </property>
         <property name="text">
          <string>generate random name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="nameLengthSB">
         <property name="toolTip">
          <string>length of the random archive name</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_13">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="compressOptionsLayout">
       <item>
        <widget class="QCheckBox" name="genPassCB">
         <property name="toolTip">
          <string>generate a random password for the archive</string>
         </property>
         <property name="text">
          <string>generate random password</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="passLengthSB">
         <property name="toolTip">
          <string>length of the random archive password</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="par2CB">
       <property name="toolTip">
        <string>generate par2</string>
       </property>
       <property name="text">
        <string>generate par2</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="delFilesCB">
       <property name="toolTip">
        <string>delete files/folders once they have been successfully posted</string>
       </property>
       <property name="text">
        <string>delete files once posted</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="postLayout">
     <item>
      <spacer name="horizontalSpacer_9">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="aboutButton">
       <property name="toolTip">
        <string>About ngPost</string>
       </property>
       <property name="text">
        <string>About</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/resources.qrc">
         <normaloff>:/icons/about.png</normaloff>:/icons/about.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_14">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="startJobsCB">
       <property name="toolTip">
        <string>start all posts when generating them</string>
       </property>
       <property name="text">
        <string>start all Posts</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="postButton">
       <property name="toolTip">
        <string>Generate Quick Posting Tabs for each file/folder</string>
       </property>
       <property name="toolTipDuration">
        <number>-5</number>
       </property>
       <property name="text">
        <string>Generate Posts</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/resources.qrc">
         <normaloff>:/icons/ngPost.png</normaloff>:/icons/ngPost.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_15">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="donateButton">
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>Donate</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/resources.qrc">
         <normaloff>:/icons/donate.png</normaloff>:/icons/donate.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_10">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SignedListWidget</class>
   <extends>QListWidget</extends>
   <header location="global">hmi/SignedListWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
